react/__tests__/components/Animation.test.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+
+import Animation from '../../Animation'
+import { renderWithIntl } from '../../__helpers__/intl-helper'
+
+describe('<Animation /> component', () => {
+  let wrapper
+
+  const renderComponent = customProps => {
+    const comp = <Animation {...customProps}> Test </Animation>
+
+    return renderWithIntl(comp)
+  }
+
+  beforeEach(() => {
+    wrapper = renderComponent()
+  })
+
+  it('should be rendered', () => {
+    expect(wrapper).toBeDefined()
+    expect(wrapper).toBeTruthy()
+  })
+
+  it('should match snapshot', () => {
```

Comment:
Shouldn't be a test for each animation type? Since each one has different styles.